### PR TITLE
adding_servers_to_terraform.yml_file_and_invertory.ini.j2_file_with_b…

### PR DESCRIPTION
**Pull request by pvisweswar**
…ionic_OS

**Ticket:** https://dimagi-dev.atlassian.net/browse/SAAS-13943

**ENVIRONMENTS AFFECTED:** Production

### CODE REVIEW

**Review by shyamkumarlchauhan — COMMENTED** [review]
I would say rather prefer to use AZ "a" as formplayer RDS "pgformplayer1-production" is in AZ "a"

**Comment by pvisweswar** [comment]
> I would say rather prefer to use AZ "a" as formplayer RDS "pgformplayer1-production" is in AZ "a"

Sure shyam, I'll change from **az: "b" to az: "a"** and update the PR then can I change the server name from **formplayer_b{i} to formplayer_a{i}.**

**Comment by pvisweswar** [comment]
> I would say rather prefer to use AZ "a" as formplayer RDS "pgformplayer1-production" is in AZ "a"

Updated the PR as per your suggestions

File: environments/production/terraform.yml [hunk 1]
```diff
@@ -260,6 +260,17 @@ servers:
     count: 21
     server_auto_recovery: true
 
+  - server_name: "formplayer_a{i}-production"
+    server_instance_type: r6a.2xlarge
+    network_tier: "app-private"
+    az: "a"
+    volume_size: 400
```
Comment (shyamkumarlchauhan) [inline environments/production/terraform.yml:267]: Change volume size to 80

Comment (pvisweswar) [reply environments/production/terraform.yml]: Updated the PR with **volume_size: 80**

**Review by shyamkumarlchauhan — APPROVED** [review]

**Review by shyamkumarlchauhan — APPROVED** [review]

**Pull request merged**